visualization/app/codeCharta/codeCharta.component.ts

```diff
@@ -33,6 +34,8 @@ export class CodeChartaController implements SettingsServiceSubscriber {
 		private settingsService: SettingsService,
 		private codeChartaService: CodeChartaService,
 		private fileStateService: FileStateService,
+		// tslint:disable-next-line
+		private nodeSearchService: NodeSearchService, // We have to inject it somewhere
```

Comment:
I don't get yet, why this has to be injected here, as it's not even used. Isn't there a nicer solution?